xarray/backends/zarr.py

```diff
@@ -10,13 +10,20 @@
 from .common import AbstractWritableDataStore, BackendArray, _encode_variable_name
 
 # need some special secret attributes to tell us the dimensions
-_DIMENSION_KEY = "_ARRAY_DIMENSIONS"
+DIMENSION_KEY = "_ARRAY_DIMENSIONS"
```

Comment:
We've successfully used `...` to represent all dimensions in `transpose` & `stack`, to the extent we could replicate that here (maybe another time though)

Reply:
This string is used as a special key in Zarr's array metadata so I think this are mostly unrelated to the selection of specific (all) dimensions.

Reply:
Ah I see, cheers